controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -392,6 +392,15 @@ void cli_bml::setFunctionsMapAndArray()
         " [<1(True) or 0(False)>] for force. Default value 1",
         static_cast<pFunction>(&cli_bml::set_wifi_credentials_caller), 2, 5, STRING_ARG, STRING_ARG,
         INT_ARG, INT_ARG, INT_ARG);
+    insertCommandToMap(
+        "bml_set_wifi_credentials_new", "<AL-MAC> <ssid> <network_key> <radio> <bss_type>",
+        "Sets new wifi credentials for the given AL-MAC."
+        "Set new ssid "
+        "Set network_key, (passphrase) "
+        "Set radio type 24g or 5g "
+        "Set bss_type, can be fronthaul, backhaul, fronthaul-backhaul",
+        static_cast<pFunction>(&cli_bml::set_wifi_credentials_new_caller), 2, 5, STRING_ARG,
```

Comment:
```suggestion
        static_cast<pFunction>(&cli_bml::set_wifi_credentials_new_caller), 5, 5, STRING_ARG,
```
It's always 5 arguments, right? None are optional?

Although it would actually be good to default radio type to 24g-5g and bss type to fronthaul.